doc/api/inspector.md

```diff
@@ -0,0 +1,72 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+<!-- YAML
+added: REPLACEME
+-->
+
+The `inspector.Session` class is a primary means of dispatching the messages to
+the V8 inspector backend and receiving message responses and notifications.
+
+An exception will be thrown if a session is created when there is already
+an ongoing inspector session (either created through this API or by connecting
+to a WebSocket port).
+
+### Event 'protocolNotification'
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification
+
+Emitted when a notification from an inspector backend was received.
+
+It is also possible to subscribe to a specific inspector notification type by
+specifying the method. E.g. this snippet will print the reason for application
+been suspended when the [Debugger.paused][debugger-paused] event is recieved
+be called whenever the program execution is suspended:
+```js
+session.on('Debugger.paused', ({reason}) => console.log(reason));
+```
+
+### session.post(method[, params][, callback])
```

Comment:
This means that `params` is optional, but you said it isn't, that a `null` has to be provided if there are no params. also, it means the callback is optional, is it?